test/Interop/SwiftToCxx/functions/swift-function-overloads.swift

```diff
@@ -10,6 +10,15 @@ public func overloadedFunc(_ y: Float) { }
 public func overloadedFuncArgLabel(x _: Int) { }
 public func overloadedFuncArgLabel(y _: Float) { }
 
+public struct IntBox { var x: CInt }
+
+public func -(lhs: IntBox, rhs: IntBox) -> CInt {
+  return lhs.x - rhs.x
+}
+
+// CHECK: class _impl_IntBox {
+
+// CHECK: int operator-(const IntBox& lhs, const IntBox& rhs) noexcept
```

Comment:
Can we also re-enable this test: `test/Interop/SwiftToCxx/functions/swift-operators.swift`?